### ISSUE TITLE
L'état d'une opération n'est pas indiqué dans la barre verticale

### ISSUE THREAD

**Issue by aymericbouzy** [aymericbouzy/balise#244]
Par soucis de cohérence dans le site, je propose qu'on déplace l'état de l'opération dans la barre verticale à gauche plutôt que d'être dans ce qui ressemble à un bouton à droite.

**Comment by Newbienathou** [comment]
Oui, et validation est intégrée dedans (opérations en attente) ?

**Comment by aymericbouzy** [comment]
On peut aussi changer l'apparence du bouton "Ajouter" pour qu'il ressemble au bouton "Modifier la répartition sur les budgets", puisqu'en réalité c'est le même bouton.
